src/sizes.js

```diff
@@ -38,21 +38,21 @@ export function getLyricsFrameSize(descriptionSize, lyricsSize) {
         lyricsUtf16Size;
 }
 
-export function getPictureFrameSize(pictureSize, mimeTypeSize, descriptionSize) {
+export function getPictureFrameSize(pictureSize, mimeTypeSize, descriptionSize, useUnicodeEncoding) {
     const headerSize = 10;
     const encodingSize = 1;
     const separatorSize = 1;
     const pictureTypeSize = 1;
     const bomSize = 2;
-    const descriptionUtf16Size = descriptionSize * 2;
+    const encodedDescriptionSize = useUnicodeEncoding ? descriptionSize * 2 : descriptionSize;
 
     return headerSize +
         encodingSize +
         mimeTypeSize +
         separatorSize +
         pictureTypeSize +
         bomSize +
-        descriptionUtf16Size +
+        encodedDescriptionSize +
```

Comment:
There's not only description size should be changed here but `bomSize` and `separatorSize` after `encodedDescriptionSize` depends on `useUnicodeEncoding` too.